data/json/npcs/isherwood_farm/NPC_Chris_Isherwood.json

```diff
@@ -167,6 +175,72 @@
       { "text": "I'd better get going.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY",
+    "dynamic_line": "[que exposition about a plan]",
+    "responses": [
+      { "text": "[go it alone]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_GO_IT_ALONE" },
+      {
+        "text": "[go with your own team]",
+        "condition": { "npc_allies": 3 },
+        "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_OWN_TEAM"
+      },
+      { "text": "[take Chris with you]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_CHRIS_JOINS" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_GO_IT_ALONE",
+    "dynamic_line": "[you quite sure? don't die]",
+    "responses": [
+      { "text": "[i'm sure]", "topic": "TALK_DONE" },
+      { "text": "[change mind, take Chris]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_CHRIS_JOINS" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_OWN_TEAM",
+    "dynamic_line": "[good you got pals]",
+    "responses": [
+      { "text": "[you want to come too?]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_CHRIS_JOINS" },
+      { "text": "[thanks bye be back with Barry]", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_CHRIS_JOINS",
+    "dynamic_line": "[yay lets kick butt. we need more?]",
+    "responses": [
+      { "text": "[nah we good trust me bro]", "effect": "follow", "topic": "TALK_DONE" },
+      { "text": "[sister come too?]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_LISA" },
+      { "text": "[i think whole family needed]", "topic": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_WHOLE_FAMILY" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_LISA",
+    "dynamic_line": "[lemme go check she prob will good idea bro]",
+    "responses": [ { "text": "[okie dokie]", "effect": { "math": [ "isherwood_rescue_lisa_coming", "=", "1" ] }, "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_TAKE_WHOLE_FAMILY",
+    "dynamic_line": "[lemme go check they prob like we all love barry.  Meet me again we plan more.]",
+    "responses": [
+      { "text": "[okie dokie]", "effect": { "math": [ "isherwood_rescue_family_coming", "=", "1" ] }, "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_ISHERWOOD_CHRIS_RESCUE_BARRY_PLAN",
+    "dynamic_line": "[Fam pull up in truck with us & we start shooting. U get inside and get Barry then we leave. Anything go wrong we let u know.  I want better gear & fortifications]",
```

Comment:
```suggestion
    "dynamic_line": "Yeah.  I think they should cover our exit.  We've got an old truck they could set up as a platform, where Dad can put down covering fire without his knee being an issue.  Eddie can drive, instead of shoot, and Luke or Mom can be ready with some first aid supplies.  Then when we get out, we've got an easy getaway set up, and there's less chance of us getting flanked while we're in there looking for Barry.\"  He looks at you, suddenly nervous.  \"Is that a good plan?",
```